src/current/v23.1/backup-architecture.md

```diff
@@ -93,6 +93,40 @@ The backup metadata files describe everything a backup contains. That is, all th
 
 With the full backup complete, the specified storage location will contain the backup data and its metadata ready for a potential [restore](restore.html). After subsequent backups of the `movr` database to this storage location, CockroachDB will create a _backup collection_. Refer to [Backup collections](take-full-and-incremental-backups.html#backup-collections) for information on how CockroachDB structures a collection of multiple backups.
 
+## Backups with locality
+
+CockroachDB supports two types of backups that work dependent on node locality. This section provides a technical overview of how these backup types work in the cluster.
```

Comment:
Maybe I'm being pedantic/bikeshedding here but "two types" seems to imply they are disjoint "types" of backup. 

I think there are a couple things I find a little weird about this:
(1) They aren't disjoint: you can use locality awareness (partitioning) and execution locality filtering individually or in combination; they're independent features.

(2) what is a "type of backup", specifically what is "a backup" in this context, as in is it the _thing_ -- the-files-in-the-bucket or is it the _process_ -- the job that writes the files? Because a locality-aware backup is indeed a distinct _thing_  in that it is partitioned, as well as a distinct process (-ish, in that it reduces network transit), but execution-locality restriction is only a distinct process. 

Perhaps these are "features" or "options" ?

Reply:
@dt  I see what you mean that that language wasn't 100% clear. I've made small edits to address this. "Features" sounds good to me. "Options" comes with the baggage of SQL options, so have avoided that. I have also tried to clarify backup job vs. backups... as I think we're generally good about using "backups" to mean your backup in storage. Or, well that is the direction the docs are heading in now.

Let me know if this has fixed what you were finding or you have other feedback here!